src/Compilers/CSharp/Test/Semantic/Semantics/SwitchTests.cs

```diff
@@ -2949,6 +2949,42 @@ public static void Main()
                 Diagnostic(ErrorCode.WRN_UnreachableCode, "break").WithLocation(14, 17));
         }
 
+        [Fact, WorkItem(47878, "https://github.com/dotnet/roslyn/issues/47878")]
+        public void Bug47878()
+        {
+            var text = @"
+using System;
+public class C
+{   
+    public static void Main()
+    {
+        int x1 = (1, 1, 1, 1, 1, 1, 1, 1, 1, 1) switch
```

Comment:
>1, 1, 1, 1, 1, 1, 1, 1, 1, 1 [](start = 18, length = 28)

Consider making these numbers increment (ie `1, 2, 3, ...`) for better readability of the test.